src/utils/Constants.tsx

```diff
@@ -7,6 +7,17 @@ import {
   VerifiedUserOutlined,
   SummarizeOutlined,
 } from '@mui/icons-material'
+import { SelectChangeEvent } from '@mui/material'
+
+export interface IMenuItems {
+  content: React.ReactNode
+  id: string
+}
+export interface Iprops {
```

Comment:
Iprops is generic, If it is related to Select, name it relevent to Select. Like "SelectProps" or "CustomSelectProps"